contracts/zaps/BaseZap.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
lets use governable instead to keep in sync with migrator

Reply:
My example is from the zapStEth strategy that I have imported and uses ownable but that makes more sense.